src/containers/Bookings.js

```diff
@@ -1,19 +1,62 @@
-import React, { Component } from 'react';
-import Search from '../components/Search.js';
-// import Results from '../components/Results.js';
-// import FakeBookings from '../data/fakeBookings.json';
+import React, { Component } from "react";
+import Search from "../components/Search.js";
+import BookingsMessage from "../components/BookingsMessage.js";
+import Results from "../components/Results.js";
+import fakeBookings from "../data/fakeBookings.json";
+import Counter from "../components/Counter";
 
 export default class Bookings extends Component {
-  search = () => {
-    console.info('TO DO!');
+  state = {
+    count: 0,
+    count2: 0,
+    sum: 0,
+    input: 0,
+    bookings: fakeBookings
   };
-  
+
+  userInput = e => {
+    e.preventDefault();
+    this.setState({
+      input: parseInt(e.target.value, 10)
+    });
+  };
+
+  bookingsById = () => {
```

Comment:
naming - I would call it `getBookingsById` or `filterBookings` (have a verb in the beginning of the name) since it's doing something (setting the state). But that is my opinion. Just be consistent, and name things the same way.

Reply:
Good suggestions, thanks!